portfolio/src/main/webapp/script.js

```diff
@@ -52,4 +52,79 @@ setInterval(() => {
     typeName();
 }, 7000);
 
-document.addEventListener("DOMContentLoaded", typeName);
\ No newline at end of file
+getComments = () => {
+    fetch('/user-comments').then(response => response.json()).then((response) => {
+        response.forEach((data) => {
+            document.getElementById('commentList').appendChild(createListElement(data));
+        })
+    })
+}
+
+createListElement = (data) => {
+    const listElement = document.createElement('li');
+    listElement.innerText = data.comment;
+    listElement.className = 'list-group-item';
+    listElement.appendChild(createDivElement(data.name, data.timestamp));
+    return listElement;
+}
+
+createDivElement = (name, timestamp) => {
+    const DivElement = document.createElement('div');
+    const nameElement = document.createElement('span');
+    nameElement.innerText = name;
+    nameElement.className = 'commentUser';
+    DivElement.appendChild(nameElement);
+    const timeStamp = document.createElement('span');
+    timeStamp.className = 'timeStamp commentUser';
+    timeStamp.innerText = timeSince(timestamp);
+    DivElement.appendChild(timeStamp);
+    return DivElement;
+}
+
+window.onload = () => {
```

Comment:
fine in this case, but note that you can accidentally override this if you a big script since this takes only one listener. Prefer addEventListener when things become complex